src/index.js

```diff
@@ -31,21 +32,24 @@ function BunnyHop (serviceName, options = {}) {
   _.defaults(options, {
     url: 'amqp://localhost',
     serialization: JsonSerialization,
-    connectionManager: DefaultConnectionManager
+    connectionManager: DefaultConnectionManager,
```

Comment:
I am not even sure we need a connectionmanager, IMO just specify the engine and the engine has a connectionmanager always

Reply:
Definitely agree, but I'm isolating this PR. there will be a future PR for ripping out the connection manager from core and into the engine